docs/src/main/asciidoc/elasticsearch-dev-services.adoc

```diff
@@ -138,13 +139,19 @@ In that case, you will need to stop and remove these containers manually.
 If you want to reuse containers for some Quarkus applications but not all of them,
 or some Dev Services but not all of them,
 you can disable this feature for a specific Dev Service by setting the configuration property
-xref:elasticsearch-dev-services.adoc#quarkus-elasticsearch-rest-client_quarkus-elasticsearch-devservices_quarkus-elasticsearch-devservices-reuse[`quarkus.elasticsearch.devservices.reuse`]
+xref:elasticsearch-dev-services.adoc#quarkus-elasticsearch-rest-client_quarkus-elasticsearch-devservices-reuse[`quarkus.elasticsearch.devservices.reuse`]
```

Comment:
the achhors for config devservices properties changed here... but I'm not sure what exactly contributed to that 😕 , or if maybe it's ok to have the new anchor moving forward ?